packages/account/src/Components/account-limits/account-limits.scss

```diff
@@ -59,13 +59,10 @@
 
         &-wrapper {
             flex: 1;
+            margin-right: 1rem;
 
-            @include desktop {
-                margin-right: 1rem;
-            }
-
-            @include mobile {
-                margin: 0 2rem 2rem;
+            @include mobile-or-tablet-screen {
+                margin: 0 1.6rem 1.6rem;
```

Comment:
@suisin-deriv , can u help with the RTL compliant CSS for this?

Reply:
I guess this should work

margin-block-start: 0;
margin-inline-start: 1.6rem;
margin-inline-end: 1.6rem;
margin-block-end: 1.6rem;

Reply:
What about this? 
margin-block: 0 1.6rem;
margin-inline: 1.6rem;